packages/backend-defaults/config.d.ts

```diff
@@ -524,6 +532,72 @@ export interface Config {
      */
     csp?: { [policyId: string]: string[] | false };
 
+    /**
+     * Rate limiting options
+     */
+    rateLimit?: {
+      /**
+       * Store to use for rate limiting. If not defined, the store will be automatically
+       * decided based on `backend.cache.store` value. If
+       * redis is not available, the store will be memory.
+       */
+      store?: 'memory' | 'redis';
+      /**
+       * Rate limiting enabled. Defaults to false.
+       */
+      enabled?: boolean;
+      /**
+       * Time frame in milliseconds or as human duration for which requests are checked/remembered.
+       * Defaults to 6000ms.
+       */
+      window?: number | HumanDuration;
+      /**
+       * The maximum number of connections to allow during the `window` before rate limiting the client.
+       * Defaults to 5.
+       */
+      limit?: number;
+      /**
+       * The response body to send back when a client is rate limited.
+       * Defaults to 'Too many requests, please try again later.'.
+       */
+      message?: JsonValue;
+      /**
+       * The HTTP status code to send back when a client is rate limited.
+       * Defaults to 429.
+       */
+      statusCode?: number;
+      /**
+       * Whether to send the legacy rate limit headers for the limit.
+       * Defaults to true.
+       */
+      legacyHeaders?: boolean;
+      /**
+       * Whether to enable support for headers conforming the RateLimit header fields for HTTP
+       * standardization. Defaults to undefined.
+       */
+      standardHeaders?: 'draft-6' | 'draft-7';
```

Comment:
What do you think about leaving out configuration for these and just setting some sensible defaults instead? In particular the status code but also the headers might simplify handling in plugin-to-plugin communication. I'm not seeing any particular reason to leave any of these configurable either